app/serializers/async_transaction_serializer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class AsyncTransactionSerializer < ActiveModel::Serializer
+  attribute :id
+  # @TODO Do we actually need to serialize the rest of these?
```

Comment:
Should we be serializing any of these attributes that the FE doesn't need?

Reply:
We need to be able to serialize the transaction id to the frontend, as they will use that to query the transaction status later on.

Reply:
I would serialize just the transaction_id and set up an id attribute similar to the other serializers (which default it to nil).